src/main.ts

```diff
@@ -111,14 +85,15 @@ const createWindow = async () => {
   w.loadURL(MAIN_WINDOW_WEBPACK_ENTRY)
 
   // Open the DevTools.
-  if (!app.isPackaged) {
-    w.webContents.openDevTools()
-  }
+  // if (!app.isPackaged) {
+  //   w.webContents.openDevTools()
+  // }
+  w.webContents.openDevTools()
```

Comment:
This will open dev tools in the prod version of the wallet

Reply:
I have removed this one to test for the build version. I have reverted it back. Sorry